tests/mocks/dependency.container.mock.ts

```diff
@@ -0,0 +1,64 @@
+import {ClassProvider,
+    DependencyContainer,
+    FactoryProvider,
+    InjectionToken,
+    RegistrationOptions,
+    TokenProvider,
+    ValueProvider
+} from "tsyringe";
+import {constructor} from "tsyringe/dist/typings/types";
+import {
+    PreResolutionInterceptorCallback,
+    PostResolutionInterceptorCallback
+} from "tsyringe/dist/typings/types/dependency-container";
+import InterceptorOptions from "tsyringe/dist/typings/types/interceptor-options";
```

Comment:
Should it be imported via "tsyringe" ?

Reply:
No because they are not exported directly. Because it's a mock, I think it's fine that we go to the type directly.